lab_5_scrapper/scrapper.py

```diff
@@ -120,19 +220,26 @@ def __init__(self, full_url: str, article_id: int, config: Config) -> None:
         """
         Initializes an instance of the HTMLParser class
         """
-        pass
+        self.full_url = full_url
+        self.article_id = article_id
+        self.config = config
+        self.article = Article(self.full_url, self.article_id)
 
     def _fill_article_with_text(self, article_soup: BeautifulSoup) -> None:
         """
         Finds text of article
         """
-        pass
+        paragraphs = article_soup.find_all('br')
+        for paragraph in paragraphs:
+            self.article.text += paragraph.text
```

Comment:
do the paragraphs always contain space at the end? Otherwise, you're joining the last word of the previous paragraph with the first word of the next one